### PR TITLE
Quickstart for linear solvers

### PR DESCRIPTION
**Pull request by mmahsereci**
# In a Nutshell
A quickstart for linear solvers. Describes basic functionality: i) how to create a test problem ii) how to run the default solver iii) what are the returned objects.

# Detailed Description
Encoding prior knowledge is not included here yet as per offline discussion. This will come in a separate tutorial (potentially before or after the interface is adapted to take selected prior information directly (see also #526 ))

# Related Issues
na

### CODE REVIEW

File: docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,2673 @@
+{
```
Comment (JonathanWenger) [inline docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb:1]: <ul><li>ProbNums's</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (JonathanWenger) [inline docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb:1]: <p>Capitalization in header: Solver</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Review by JonathanWenger — APPROVED** [review]

File: docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb (hunk 1 above)
Comment (mmahsereci) [reply docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb:1]: <p>done</p>

Comment (mmahsereci) [reply docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb:1]: <p>done</p>

**Pull request merged**